### DISCUSSION TITLE
not working with vestacp

### DISCUSSION THREAD

**Discussion by raaaas** [angristan/wireguard-install#193] [Help]
hello, thanks for great script,
 i have 2 vps  

1- Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-126-generic x86_64) vps from OVH with VESTACP
2-  Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-126-generic x86_64) vps from  NL without VESTACP

in server 2 i just git cloned script and everything working fine but in first server not passing over handshake. 

server 1 tcpdump :
tcpdump -n -v -i wg2
tcpdump: listening on wg2, link-type RAW (Raw IP), capture size 262144 bytes
-------------
tcpdump -n -v -i eth0 port 1010
tcpdump: listening on eth0, link-type EN10MB (Ethernet), capture size 262144 bytes
-------------

 wg-quick up wg2
[#] ip link add wg2 type wireguard
[#] wg setconf wg2 /dev/fd/63
[#] ip -4 address add 10.66.66.1/24 dev wg2
[#] ip -6 address add fd42:42:42::1/64 dev wg2
[#] ip link set mtu 1420 up dev wg2
[#] iptables -A FORWARD -i eth0 -o wg2 -j ACCEPT; iptables -A FORWARD -i wg2 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE; ip6tables -A FORWARD -i wg2 -j ACCEPT; ip6tables -t nat -A POSTROUTING -o eth0 -j MASQUERADE

**Comment by layanto** [comment]
Not sure if related to this issue. I use HeatiaCP on Ubuntu. Immediately after installing WireGuard or adding new client, WireGuard works - I can connect to server and internet through the server works. But if I restart the server, WireGuard stops working. I can still connect to server but internet is not working. I found that I had to restart wg-quick@wg0 service and then everything would work again. My work around fix is to delay the start of wg-quick@wg0 service 5min after server starts to ensure this service starts after all other HestiaCP services have started. Seems to work now.
